src/axolotl/utils/tokenization.py

```diff
@@ -62,7 +62,7 @@ def process_tokens_for_rl_debug(tokens, color, tokenizer, text_only):
     """Helper function to process and color tokens."""
     colored_tokens = [
         color_token_for_rl_debug(tokenizer.decode(token), token, color, text_only)
-        for token in tokenizer.encode(tokens)
+        for token in tokenizer.encode(tokens, add_special_tokens=False)
```

Comment:
Note: I added this since by default I saw that this step was including the bos token all the time.  Since that's already included it seemed reasonable to not add it in a second time.